earth_enterprise/src/fusion/autoingest/sysman/AssetVersionD.cpp

```diff
@@ -230,16 +250,28 @@ void
 AssetVersionImplD::PropagateStateChange(void)
 {
   notify(NFY_VERBOSE, "PropagateStateChange(%s): %s",
```

Comment:
I would probably change this notify level to NFY_PROGRESS so that when we are logging at just the NFY_PROGRESS level we get the context of which asset is propagating its state change